source/linear-algebra/exercises/outcomes/EV/EV1/template.xml

```diff
@@ -46,8 +46,8 @@ The vector equation has no solutions, so <m>{{v}}</m> is not a linear combinatio
         <knowl>
             <content>
                 <p>
-If your statement was true, describe a linear combination of <m>{{veclist}}</m>
-that equals <m>{{v}}</m>.
+If your statement was true, describe how to use your RREF matrix to construct
+a linear combination of <m>{{veclist}}</m> that equals <m>{{v}}</m>.
```

Comment:
```suggestion
If your statement was true, explain and demonstrate how to construct
a specific linear combination of <m>{{veclist}}</m> that equals <m>{{v}}</m>.
```

Maybe an okay compromise @jkostiuk @siwelwerd ?

Reply:
I like it!